src/Compilers/CSharp/Portable/BoundTree/BoundNodes.xml

```diff
@@ -396,6 +396,12 @@
 
     <Field Name="Left" Type="BoundTupleExpression" Null="disallow"/>
     <Field Name="Right" Type="BoundConversion" Null="disallow"/>
+    <Field Name="IsUsed" Type="Boolean" Null="NotApplicable"/>
+  </Node>
+
+  <Node Name="BoundVoid" Base="BoundExpression">
+    <!-- Non-null type is required for this node kind, but it will always be the void type  -->
```

Comment:
I'll remove the comment. We'll be using a type that is not "void".
@cston I think this reinforces your discomfort with calling this node BoundVoid. I'll try to think of another name or approach.